sources/editor/Xenko.Assets.Presentation/AssetEditors/Gizmos/ScaleGizmo.cs

```diff
@@ -125,13 +125,13 @@ private float ComputeScaleFactorFromTranslation(float translation)
         {
             const float MaxScaleValue = 1f / MathUtil.ZeroTolerance;
 
-            // snap the value if needed BEFORE the exp function
+            // snap the value if needed BEFORE the scale function
             if (UseSnap)
                 translation = MathUtil.Snap(translation, SnapValue);
 
-            var scaleValue = translation > 0 ? (float)Math.Exp(translation) : 1 / ((float)Math.Exp(-translation));
+            //var scaleValue = translation > 0 ? (float)Math.Exp(translation) : 1 / ((float)Math.Exp(-translation));
```

Comment:
You can omit commented-out lines from PRs, we can always rollback if necessary.

Reply:
This was for me, I just forgot to remove it